engines/download_only.py

```diff
@@ -124,11 +136,11 @@ def dummy_method(self, *args, **kwargs):
                 }
 remove_methods = ['insert_data_from_file']
 for name, method in methods:
-    if (not name in keep_methods
-        and not 'download' in name
-        and not 'file' in name
-        and not 'dir' in name):
-
+    if (
+            name not in keep_methods and
+            'download' not in name and
+            'file' not in name and
+            'dir' not in name):
         setattr(engine, name, dummy_method)
 for name in remove_methods:
-    setattr(engine, name, dummy_method)
+    setattr(engine, name, dummy_method)
```

Comment:
Is getting rid of the newline at the end here intentional? Seems like most of our files end in a newline.